modules/vba_documents/app/services/vba_documents/slack/messenger.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+
+module VBADocuments
+  module Slack
+    class Messenger
+      ALERT_URL = Settings.vba_documents.slack.default_alert_url
```

Comment:
This differs from the `AppealsAPI` implementation in that it uses a Slack webhook rather than the combination of the `https://slack.com/api/chat.postMessage`, Slack channel ID, and an API token. If we want to bring it in line with how Appeals does it, I can create a ticket for doing that separately. (Will require changes to the `devops` repo.)

Reply:
Will be making the change above in my next PR.